### ISSUE TITLE
[Backport][v2.9] [BUG] Unable to create the CIS scan output because of the configMap size is exceeding 1 MB

### ISSUE THREAD

**Issue by vardhaman22** [rancher/rancher#47606]
backport: https://github.com/rancher/rancher/issues/47296

**Comment by vivek-shilimkar** [comment]
Validated on rancher v2.10-head

- Created RKE2 harden cluster
- Installed CIS released version 6.3.0, and ran v1.8 permissive profile scan on all above clusters.
- Downloaded cluster scan report yaml.
- Upgraded chart to 6.4.0-rc.1 version.
- Ran the v1.8 permissive profile scans and all passed successfully.
- Also downloaded cluster scan report yaml.
- Compared the scan report yaml size and it's significantly reduced in CIS 6.4.0-rc.1 version.

Hence this issue can be closed.

**Comment by vardhaman-surana** [comment]
we will need to mention this command in the docs 
```
export REPORT="<scan-report-name>"
kubectl get clusterscanreport $REPORT -o json |jq ".spec.reportJSON | fromjson" | jq -r ".actual_value_map_data" | base64 -d | gunzip | jq .
```
